frontend/src/component/layout/MainLayout/MainLayout.tsx

```diff
@@ -17,29 +14,63 @@ import { ConditionallyRender } from 'component/common/ConditionallyRender/Condit
 import { useChangeRequestsEnabled } from 'hooks/useChangeRequestsEnabled';
 import { DraftBanner } from './DraftBanner/DraftBanner';
 
-const useStyles = makeStyles()(theme => ({
-    container: {
-        height: '100%',
-        justifyContent: 'space-between',
+interface IMainLayoutProps {
+    children: ReactNode;
+}
+
+const MainLayoutContainer = styled(Grid)(() => ({
+    height: '100%',
+    justifyContent: 'space-between',
+}));
+
+const MainLayoutContentWrapper = styled('main')(({ theme }) => ({
+    margin: '0 auto',
+    flex: 1,
+    width: '100%',
+    backgroundColor: theme.palette.contentWrapper,
+    position: 'relative',
+}));
+
+const MainLayoutContent = styled(Grid)(({ theme }) => ({
+    width: '1250px',
+    marginLeft: 'auto',
+    marginRight: 'auto',
+    [theme.breakpoints.down('lg')]: {
+        width: '1024px',
+    },
+    [theme.breakpoints.down(1024)]: {
+        width: '100%',
+        marginLeft: 0,
+        marginRight: 0,
     },
-    contentContainer: {
-        height: '100%',
-        padding: '3.25rem 0',
-        position: 'relative',
-        [theme.breakpoints.down('md')]: {
-            padding: '3.25rem 0.75rem',
-        },
+    [theme.breakpoints.down('sm')]: {
+        minWidth: '100%',
     },
 }));
 
-interface IMainLayoutProps {
-    children: ReactNode;
-}
+const StyledImg = styled('img')(() => ({
+    display: 'block',
+    position: 'fixed',
+    zIndex: 0,
+    bottom: 0,
+    right: 0,
+    width: 400,
+    pointerEvents: 'none',
+    userSelect: 'none',
+}));
+
+const MainLayoutContentContainer = styled('div')(({ theme }) => ({
+    height: '100%',
+    padding: '3.25rem 0',
+    position: 'relative',
+    [theme.breakpoints.down('md')]: {
+        padding: '3.25rem 0.75rem',
```

Comment:
We should use theme.spacing here

Reply:
In essence, with theme spacing you just double the REM value. So this would be `theme.spacing(6.5, 1.5)`